File: Pratica3/SistemaViagem/src/dados/Cliente.java
package src.dados;

public class Cliente {

    private long cpf;
    private String nome;
    private String endereco;
    private long telefone;
    private int quantReservas = 0;
    private Reserva [] reservas = new Reserva[100];

    public long getCpf() {
        return cpf;
    }

    public void setCpf(Long cpf) {
        this.cpf = cpf;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getEndereco() {
        return endereco;
    }

    public void setEndereco(String endereco) {
        this.endereco = endereco;
    }

    public long getTelefone() {
        return telefone;
    }

    public void setTelefone(long telefone) {
        this.telefone = telefone;
    }

    public Reserva [] getReservas() {
        return reservas;
    } 

    public void setReservas(Reserva[] reservas) {
        this.reservas = reservas;
    }

    public void reservarIda(Reserva reserva){
        if(quantReservas == reservas.length){
            System.out.println("Reservas cheias!!!");
        }else{
            reservas[quantReservas] = reserva;
            quantReservas++;
        }
    }

    public void reservarVolta(Reserva ida, Reserva volta){
        
        for(int i = 0; i < quantReservas ; i++){
            if(reservas [i] == ida ){
                reservas[i].setVolta(volta);
                reservas[quantReservas] = volta;
                quantReservas++;
            }
        }
    
    }

    public String toString(){
        return "CPF: "+cpf+"\nNome: "+nome+"\nEndereco: "+endereco+"\nTelefone: "+telefone;
    }
}

File: Listas/Lista1/ex5/dados/Cliente.py
import string
from Endereco import Endereco
from Veiculo import Veiculo

class Cliente:

    def __init__(self):
        self.__nome = ""
        self.__cpf = ""
        self.__endereco = Endereco()
        self.__veiculo = Veiculo()

    def getNome(self):
        return self.__nome

    def setNome(self, nome: string):
        self.__nome = nome

    def getCpf(self):
        return self.__cpf

    def setCpf(self, cpf : string):
        self.__cpf = cpf

    def getEndereco(self):
        return self.__endereco

    def setEndereco(self, endereco: Endereco):
        self.__endereco = endereco

    def getVeiculo(self):
        return self.__veiculo

    def setVeiculo(self, veiculo : Veiculo):
        self.__veiculo = Veiculo

    def __str__(self):
        return 'Nome: {} \n Cpf: {} \n Endereco: {} \n Veiculo Alugado: {}'.format(self.__nome, self.__cpf, self.__endereco, self.__veiculo)
